--- a/scripts/xinitrc.in.in
+++ b/scripts/xinitrc.in.in
@@ -76,7 +76,7 @@ test -r "$BASEDIR/Xcursor.xrdb" && XRESO
 test -r "$HOME/.Xresources" && XRESOURCES="$XRESOURCES $HOME/.Xresources"
 
 # load all X resources (adds /dev/null to avoid an empty list that would hang the process)
-cat /dev/null $XRESOURCES | xrdb -nocpp -merge -
+cat /dev/null $XRESOURCES | xrdb -merge -
 
 # load local modmap
 test -r $HOME/.Xmodmap && xmodmap $HOME/.Xmodmap
